### PR TITLE
Align Skeleton to skeleton/pose share code

### PR DESCRIPTION
**Pull request by tomer-algo**
# Intro
  - Align Skeleton to skeleton/pose share code
# Project
  - Skeleton
  - Pose
# Implemented Sections
  - Use functions from code base s3://nanit-algo-resources/algo-infra/package-versions/nanit_algo_infra_online_additions-1.1.1-py3-none-any.whl
  - On which modules we can see the effect of the change - Utils Functions + Split
# Tests Description
  - Training
  - Split
  - Run Inference on TorchServe (local)
# Undeveloped Sections
  - 
# What's New
  - Using Code-base
# Checklist
  - [x] Module tested fully after last commit
  - [ ] Scheduled a PR meeting (if needed)
  - [ ] Documentation (comments or a dedicated document)
### Tested platforms
  - [x] Local
  - [ ] Cloud (SageMaker)
  - [ ] Staging
  - [ ] Not relevant
### Tested Cameras
  - [ ] Gen 1.5
  - [ ] Gen 2
  - [x] Not relevant
# Comments
### Included Features
  - 
### Infrastructure Dependencies
  - s3://nanit-algo-resources/algo-infra/package-versions/nanit_algo_infra_online_additions-1.1.1-py3-none-any.whl
### Tech Comments
  - 
### Testing Instructions
  - 
### Todos
  -

### CODE REVIEW

**Review by felixNan — APPROVED** [review]

File: analysis/analysis_utils.py [hunk 1]
```diff
@@ -96,12 +95,6 @@ def distance_two_points(p1, p2):
     return np.sqrt(dx**2 + dy**2)
 
 
-def init_wandb():
-    os.environ['WANDB_API_KEY'] = get_weights_and_biases_owner()
-    os.environ['WANDB_DISABLE_CODE'] = '*.patch'
```
Comment (Nimrods) [inline analysis/analysis_utils.py:101]: Did you change this command to the updated one W&B shared with Felix few months ago?
(in the shared code package I mean)

File: export/config.properties [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
-    "1.0": {\
+    "1.2": {\
         "defaultVersion": true,\
-        "marName": "skeleton.mar",\
+        "marName": "skeleton1.2.mar",\
         "minWorkers": 1,\
         "maxWorkers": 1,\
-        "batchSize": 2,\
-        "maxBatchDelay": 5000,\
+        "batchSize": 1,\
```
Comment (Nimrods) [inline export/config.properties:12]: why just 1?
low throughput?

**Review by Nimrods — CHANGES_REQUESTED** [review]
Skeleton derived class shouldn't contain the base class same functionality only what's need to override

File: analysis/analysis_utils.py (hunk 1 above)
Comment (tomer-algo) [reply analysis/analysis_utils.py:101]: using this function from `nanit-algo-infra-online-additions`
```
def init_wandb(project, notes, job_type):
    os.environ['WANDB_API_KEY'] = get_weights_and_biases_owner()
    wandb.init(project=project, notes=notes, job_type=job_type)

```

File: export/config.properties (hunk 2 above)
Comment (tomer-algo) [reply export/config.properties:12]: low throughput, I aligned the code to the deployed model

File: skeleton_split/skeleton_split.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+from tqdm import tqdm
+
+from lib.nanit_utils.utils_functions import bbox_to_center_scale
+
+from python_tools import Constants, S3Handler
+from python_tools.Split import SkeletonBaseSplitModule
+from python_tools.BBox import BBox
+
+
+class SkeletonSplitModule(SkeletonBaseSplitModule):
```
[the remark this answers is not in the document]
Comment (tomer-algo) [reply skeleton_split/skeleton_split.py:12]: I didn't defined the functionality again, I've added another level in OOP
in  `nanit-algo-infra-online-additions` I have
`BaseSplitModule` -> `SkeletonBaseSplitModule`
for example in `SkeletonBaseSplitModule` I have additional method `parse_joints_from_skeleton_mark` that relevant only for "Skeleton Based" model

File: analysis/analysis_utils.py (hunk 1 above)
Comment (Nimrods) [reply analysis/analysis_utils.py:101]: So we upload the code everytime, tnx

**Review by Nimrods — APPROVED** [review]

**Pull request merged**